contracts/protocol/facets/DisputeHandlerFacet.sol

```diff
@@ -226,20 +231,11 @@ contract DisputeHandlerFacet is DisputeBase, IBosonDisputeHandler {
      * - Dispute was escalated and escalation period has elapsed
      *
      * @param _exchangeId  - the id of the associated exchange
-     * @param _buyerPercent - percentage of the pot that goes to the buyer
-     * @param _sigR - r part of the signer's signature
-     * @param _sigS - s part of the signer's signature
-     * @param _sigV - v part of the signer's signature
+     * @param _percent - percentage of the pot that goes to the buyer or seller
      */
-    function resolveDispute(
-        uint256 _exchangeId,
-        uint256 _buyerPercent,
-        bytes32 _sigR,
-        bytes32 _sigS,
-        uint8 _sigV
-    ) external override disputesNotPaused nonReentrant {
+    function resolveDispute(uint256 _exchangeId, uint256 _percent) external override disputesNotPaused nonReentrant {
```

Comment:
As mentioned in the [BPIP discussion](https://github.com/bosonprotocol/BPIPs/pull/1#issuecomment-1366135303), changing this method:

1) represents a breaking change that would require a major version rev to 3.0.0
2) discards the signature, which is required to ensure that the two parties are in accord.

It could be added as an alternative method with a name like `resolveDisputeWithProposal` or something, but it's not clear why this is even part of this proposed implementation. The BPIP is about using Kleros to handle escalations. This comes prior to escalation and is not a prerequisite for such.

Reply:
Thanks for the comment @cliffhall. I am going to update the BPIP so we don't introduce the breaking change.

A new method is needed because Kleros requires two options to choose from. It means that escalation only makes sense when a buyer and a seller have proposals that don't match, like 50% and 70%.

A state of those proposals must be stored somewhere before the buyer escalates it to a DR.